configuration/analyzer_config.json

```diff
@@ -450,6 +450,38 @@
       }
     }
   },
+  "CyberChef": {
+    "type": "observable",
+    "python_module": "cyberchef.CyberChef",
+    "description": "scan a string against CyberChef",
+    "disabled": false,
+    "observable_supported": [
+      "generic"
+    ],
+    "external_service": true,
+    "config": {
+      "soft_time_limit": 50,
+      "queue": "long"
+    },
+    "secrets": {},
+    "params": {
+      "predefined_recipe_name": {
+        "value": "to decimal",
+        "type": "str",
+        "description": "Name of pre-defined recipe to use."
+      },
+      "custom_recipe": {
+        "value": [],
+        "type": "list",
+        "description": "Custom recipe to use (instead of a predefined one). NOTE: This is only read if predefined_recipe_name is blank"
```

Comment:
If we put a single field for a recipe and a string is entered, it will be difficult for us to judge is the user intended this string as "name of predefined recipe" or as the content of an actual recipe

Reply:
Ok now I understood the use case. Makes sense but I think we should rename these fields.

`predefined_recipe_name` could be `recipe_name` while `custom_recipe` could be `recipe_code`

Reply:
we could also add a link here as a reference for examples: https://github.com/mattnotmax/cyberchef-recipes